docs/data/material/components/accordion/ControlledAccordions.js

```diff
@@ -3,8 +3,29 @@ import Accordion from '@mui/material/Accordion';
 import AccordionDetails from '@mui/material/AccordionDetails';
 import AccordionSummary from '@mui/material/AccordionSummary';
 import Typography from '@mui/material/Typography';
+import { styled } from '@mui/material/styles';
 import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
 
+const SummaryText = styled('span')(({ theme }) => ({
+  margin: 0,
+  textAlign: 'left',
+  variants: [
+    {
+      props: { secondary: true },
+      style: {
+        color: theme.palette.text.secondary,
+      },
+    },
+    {
+      props: ({ secondary }) => !secondary,
+      style: {
+        width: '33%',
+        flexShrink: 0,
+      },
+    },
+  ],
+}));
```

Comment:
I think having two different components would make the demo easier to follow.

```suggestion
const SummaryLabel = styled('span')(({ theme }) => ({
  width: '33%',
  flexShrink: 0,
}));

const SummaryText = styled('span')(({ theme }) => ({
  color: theme.palette.text.secondary,
}));
```

Reply:
Makes sense. Done.